src/main/java/tutorlink/commandpackage/GetGradeStudentCommand.java

```diff
@@ -0,0 +1,14 @@
+package tutorlink.commandpackage;
+
+import tutorlink.resultpackage.CommandResult;
+
+public class GetGradeStudentCommand extends Command {
+
+    public GetGradeStudentCommand() {
+    }
+
+    @Override
+    public CommandResult execute() {
+        return new CommandResult("Needed new implementation");
+    }
+}
```

Comment:
What does this command do?

Reply:
Calculate the final GPA of the student for the course, although due to the old implementation relating this to course, I temporarily put "Needed new implementation" for now